plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Definition.hs

```diff
@@ -83,7 +83,7 @@ test_Const =
         b <- forAll Gen.bool
         let tC = mkConstant () c
             tB = mkConstant () b
-            text = toTypeAst @_ @DefaultUni @Text Proxy
+            text = toTypeAst @_ @_ @DefaultUni @Text Proxy
```

Comment:
> Is there a specific reason why we have typed terms in tests in untyped-plutus-core?

I think in general it's largely accidental with the whole untyped-plutus-core depending on typed plutus-core situation, but I think in this specific module I just wanted to make sure that things evaluate correctly and wired in the type checker so that there are no surprises and to help with debugging.